pyuvdata/uvdata/initializers.py

```diff
@@ -151,7 +153,7 @@ def get_time_params(
 
 
 def get_freq_params(
-    freq_array: np.ndarray, channel_width: float | np.ndarray | None = None
+    *, freq_array: np.ndarray, channel_width: float | np.ndarray | None = None
```

Comment:
For a function like this, I feel like `freq_array` is an obvious first argument. Along with the fact it is the only required parameter, would it be fine to leave as a positional argument?

Reply:
yep, sounds good.